docs/src/running-validator/validator-reqs.md

```diff
@@ -2,35 +2,31 @@
 title: Validator Requirements
 ---
 
-## Hardware
-
-- CPU Recommendations
-  - We recommend a CPU with the highest number of cores as possible. AMD Threadripper or Intel Server \(Xeon\) CPUs are fine.
-  - We recommend AMD Threadripper as you get a larger number of cores for parallelization compared to Intel.
-  - Threadripper also has a cost-per-core advantage and a greater number of PCIe lanes compared to the equivalent Intel part. PoH \(Proof of History\) is based on sha256 and Threadripper also supports sha256 hardware instructions.
-- SSD size and I/O style \(SATA vs NVMe/M.2\) for a validator
-  - Minimum example - Samsung 860 Evo 2TB
-  - Mid-range example - Samsung 860 Evo 4TB
-  - High-end example - Samsung 860 Evo 4TB
+## Hardware Recommendations
+
+- CPU
+  - 12 cores / 24 threads, or more
+  - 2.8GHz, or faster
+  - AVX2 instruction support
+  - Support for AVX512f and/or SHA-NI instructions is helpful
+  - The AMD Threadripper Zen3 series is popular with the validator community
+- RAM
+  - 128GB, or more
+  - Motherboard with 256GB capacity suggested
+- Disk
+  - PCIe gen3 x4 NVME SSD, or better
+  - Accounts: 500GB, or larger. High TBW
```

Comment:
TBW = total band width? is it common?

Reply:
total bytes written, it's a common acronym related to SSD drive life.

Reply:
I'll add a parenthetical spelling it out to the first use